test/e2e/webhook_test.go

```diff
@@ -0,0 +1,82 @@
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+
+)
+
+var _ = Describe("bundle api validating webhook", func() {
+	When("Bundle is valid", func() {
+		var (
+			bundle *rukpakv1alpha1.Bundle
+			ctx    context.Context
+		)
+		AfterEach(func() {
+			By("deleting the testing Bundle resource")
+			err := c.Delete(ctx, bundle)
+			Expect(err).To(BeNil())
+		})
+
+		It("should create the bundle resource", func() {
+			ctx = context.Background()
+
+			bundle = &rukpakv1alpha1.Bundle{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "valid-bundle-",
+				},
+				Spec: rukpakv1alpha1.BundleSpec{
+					ProvisionerClassName: plainProvisionerID,
+					Source: rukpakv1alpha1.BundleSource{
+						Type: rukpakv1alpha1.SourceTypeImage,
+						Image: &rukpakv1alpha1.ImageSource{
+							Ref: "testdata/bundles/plain-v0:valid",
+						},
+					},
+				},
+			}
+			err := c.Create(ctx, bundle)
+			Expect(err).To(BeNil())
+		})
+	})
```

Comment:
Do we need a test case for valid Bundles? Is there too much overlap with the core testing suite?

Reply:
I see.  I take this out.

Reply:
I actually think this is valuable enough to leave it. It does overlap with the plain provisioner tests, but it probably makes sense to have positive and negative tests for the webhook logic even if there is overlap with the provisioner e2es.